bin/web.js

```diff
@@ -40,7 +40,8 @@ fs.readFile(path.join(__dirname, '../config/config.json'), 'utf8', (err, data) =
 
     const pictureOptims = (pictureSizes || [700, 350]).reduce((optims, size, i) => {
       const label = i === 0 ? 'big' : i === 1 ? 'normal' : 'small'
-      optims.push({ size, label, webp: true })
+      optims.push({ size, label, webp: true, avif: false })
+      optims.push({ size, label, webp: false, avif: true })
```

Comment:
```suggestion
      optims.push({ size, label, avif: false })
      optims.push({ size, label, avif: true })
```

não é um problema, mas acho que a gente pode tirar o `webp`, se não avif, é webp...

Reply:
OK